content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -2332,6 +2332,114 @@ Starting in v1.16, this annotation was removed in favor of
 [Pod Priority](/docs/concepts/scheduling-eviction/pod-priority-preemption/).
 {{< /note >}}
 
+### jobset.sigs.k8s.io/jobset-name
+
+Type: Label
```

Comment:
All of these are used as both labels and annotations (except the taint).

Reply:
okay i will register for both labels and annotation